app/javascript/admin/controllers/modal_controller.js

```diff
@@ -1,13 +1,27 @@
 import { Controller } from 'stimulus';
```

Comment:
I looked through the codebase and can't find any instances of markup that has a `data-controller="modal"`. And I tried some admin pages and the only modal that appears to pop comes from `app/javascript/admin/adminModal.js`.

I wonder if this is dead code? Pinging @msarit as she's been in this part of the codebase recently.

Reply:
Thanks for the ping, @nickytonline. From what I know, this `modal_controller` code is not in use. I'll take a closer look in the morning. 🌙

Reply:
@nickytonline @msarit it's used in the admin sections:

- Add navigation link `_add_navigation_link_modal`
- Edit navigation link `_edit_navigation_link_modal`

And the edit profile fields section that I mention in the PR that I can't find a use of in the app (so I think might be dead?):

- `_add_group_modal`
- `_add_profile_field_modal`
- `-edit_group_modal`

Reply:
@aitchiss can you expand more on 
> And the edit profile fields section that I mention in the PR that I can't find a use of in the app (so I think might be dead?):
>_add_group_modal
>_add_profile_field_modal
>-edit_group_modal

I worked on this initially and I may be able to help, can you point me to the code or classname that you're try figure out exists or not?

Reply:
Thanks for your help @Ridhwana!

The modal controller is referenced in the erb files: `_add_group_modal.html.erb` and the others noted above. These files are all rendered in `app > views > admin > profile_fields > index.html.erb`

But I haven't been able to find where in the app UI we actually link/present these views to users

Reply:
Just summarizing our slack thread for anyone following along:
The reason that @aitchiss was not seeing the profile admin pages on the UI was because it's behind a feature flag. We've enabled the feature flag and we're able to see it now :)